cds/modules/records/static/templates/cds_records/detail.default.html

```diff
@@ -9,27 +9,94 @@
         </ul>
       </div>
       <div class="cds-record-detail-bottom">
+        <!-- Title -->
         <h1>{{ record.title_statement.title || 'No title :(' }}</h1>
-        <ul class="list-inline record-detail-metadata-collection">
-          <li ng-show="record.collections.length >0" >
-            <i class="text-primary fa fa-tags"></i>
-            <span ng-repeat="col in record.collections">
-              <span class="label label-warning">{{ col.primary }}</span>
+        <!-- Authors -->
+        <div class="row" ng-show="record.main_entry_personal_name || record.added_entry_personal_name.length > 0" >
+          <div class="col-xs-3 col-sm-2">
+            <p>Authors:</p>
+          </div>
+          <div class="col-xs-9 col-sm-10">
+            <span ng-show="record.main_entry_personal_name">
+              <a href="/search?authors={{record.main_entry_personal_name.personal_name}}"> {{ record.main_entry_personal_name.personal_name }}</a>
             </span>
-          </li>
-          <li ng-show="record.main_entry_personal_name.length > 0">
-            <i class="fa text-primary" ng-class="{'fa-user': record.main_entry_personal_name.length === 1, 'fa-users': record.main_entry_personal_name.length > 1}" ></i>
-            <span ng-repeat="user in record.main_entry_personal_name">
-              {{ user.personal_name }}
+            <span ng-repeat="user in record.added_entry_personal_name">
+              , <a href="/search?authors={{record.main_entry_personal_name.personal_name}}">
```

Comment:
👍 could you please ``ng-href`` instead of ``href``? [angular|https://docs.angularjs.org/api/ng/directive/ngHref] :P